metaflow_extensions/ray/plugins/ray_log_tailer.py

```diff
@@ -0,0 +1,96 @@
+import os
+import time
+import glob
+import threading
+from pathlib import Path
+
+
+class RayLogTailer:
+    """
+    Tails Ray log files and streams them to stdout so they appear in CloudWatch logs.
```

Comment:
```suggestion
    Tails Ray log files and streams them to stdout.  
```

Avoiding cloud specific service name since Metaflow is multi-cloud.

Reply:
Done